src/pages/workspaces/workspace/workflows/DataStepContent.js

```diff
@@ -86,69 +94,115 @@ export default class DataStepContent extends Component {
     return h(Modal, {
       title: 'Select Data',
       okButton: h(ButtonPrimary, {
-        tooltip: isChooseSets && _.size(selectedEntities) > 10 && 'Please select 10 or fewer sets',
+        tooltip: Utils.cond(
+          [isChooseSets && _.size(selectedEntities) > 10, () => 'Please select 10 or fewer sets'],
+          [!_.size(selectedEntities), () => 'Please select data']
+        ),
         disabled: !!errors || !this.isValidSelectionModel(),
         onClick: () => onSuccess(newSelectionModel)
       }, 'OK'),
       onDismiss,
       width: 'calc(100% - 2rem)'
     }, [
-      div({ style: { ...Style.elements.sectionHeader, marginBottom: '1rem' } },
-        [`Select ${(isSet ? 'up to 10 ' : '') + rootEntityType}s to process${isSet ? ' in parallel' : ''}`]),
       div({
         style: {
           padding: '1rem 0.5rem', lineHeight: '1.5rem'
         }
       }, [
-        !isSet && h(div, { role: 'radiogroup', 'aria-label': 'Select entities' }, [
-          div([
-            h(RadioButton, {
-              text: `Process all ${count} rows`,
-              name: 'process-rows',
-              checked: isProcessAll,
-              onChange: () => this.setNewSelectionModel({ type: processAll, selectedEntities: {} }),
-              labelStyle: { marginLeft: '0.75rem' }
-            })
-          ]),
-          div([
-            h(RadioButton, {
-              text: 'Choose specific rows to process',
-              name: 'process-rows',
-              checked: isChooseRows,
-              onChange: () => this.setNewSelectionModel({ type: chooseRows, selectedEntities: {} }),
-              labelStyle: { marginLeft: '0.75rem' }
-            })
+        div({ role: 'radiogroup', 'aria-label': 'Select entities' }, isSet ?
```

Comment:
Thinking about this, I think I'd prefer a small change:

rather than 
```js
isSet ? [] : []
```
I'd like 
```js
[isSet ? h(Fragment, []) : h(Fragment, [])]
```